setup.py

```diff
@@ -47,13 +47,18 @@ def package_files(package_dir, subdirectory):
         'scipy',
         'astropy',
         'pyuvdata',
-        'aipy>=3.0rc2',
         'pyephem',
         'uvtools @ git+git://github.com/HERA-Team/uvtools',
         'linsolve @ git+git://github.com/HERA-Team/linsolve',
         'hera_qm @ git+git://github.com/HERA-Team/hera_qm',
         'scikit-learn'
     ],
+    'extras_require': {
+        "all": [
+            'aipy>=3.0rc2',
+        ]
+    }
```

Comment:
I think we need a trailing comma on this line to fix the installation issues.

Reply:
yeah... :man_facepalming: